controllers/sms-gateway.js

```diff
@@ -73,9 +80,11 @@ function getOutgoing(callback) {
         content: message.message,
       };
     });
-    markMessagesForwarded(messages, function() {
+    // TODO add this back in once this function also queries for
+    // messages in state: 'forwarded-to-gateway'
+    // markMessagesForwarded(messages, function() {
```

Comment:
Gateway doesn't call the messages API. Any changes made to the gateway code are complicated because we either need to worry about backwards compatibility or try to sync updates. This is a quick fix to get 0.4 up and running, and we can do a much more robust solution in 2.10 in #3073